make.ps1

```diff
@@ -18,17 +18,74 @@ function FindMSBuild
 	return $null
 }
 
+function CopyDependencies
+{
+	cd thirdparty
+	./fetch-thirdparty-deps.ps1
+	cp *.dll ..
+	cp windows/*.dll ..
+	cd ..
+	echo "Dependencies copied."
+
+	return $LASTEXITCODE
+}
+
+function TestModsAndMaps
+{
+	if(Test-Path ./OpenRA.Utility.exe)
+	{
+		echo "Testing mods..."
+		if(Test-Path ./mods/ra/OpenRA.Mods.RA.dll)
+		{
+			echo ""
+			echo "Testing Red Alert mod MiniYAML..."
+			./OpenRA.Utility.exe ra --check-yaml
+	
+			if(Test-Path ./mods/cnc/OpenRA.Mods.Cnc.dll)
+			{
+				echo ""
+				echo "Testing Tiberian Dawn mod MiniYAML..."
+				./OpenRA.Utility.exe cnc --check-yaml
+			}
+
+			if(Test-Path ./mods/d2k/OpenRA.Mods.D2k.dll)
+			{
+				echo ""
+				echo "Testing Dune 2000 mod MiniYAML..."
+				./OpenRA.Utility.exe d2k --check-yaml
+			}
+
+			if(Test-Path ./mods/ts/OpenRA.Mods.TS.dll)
+			{
+				echo ""
+				echo "Testing Tiberian Sun mod MiniYAML..."
+				./OpenRA.Utility.exe ts --check-yaml
+			}
+		}
+		else
+		{
+			echo "Error: Red alert is needed to work properly!"
+			return 1
+		}
+	}
+	else
+	{
+		echo "Error: OpenRA.Utility.exe is missing."
+		return 1
+	}
+}
+
 if ($args.Length -eq 0)
 {
 	echo "Command list:"
 	echo ""
-	echo "  all             Builds the game and its development tools."
+	echo "  all             Copies dependencies, builds the game and its development tools."
 	echo "  dependencies    Copies the game's dependencies into the main game folder."
 	echo "  version         Sets the version strings for the default mods to the latest"
 	echo "                  version for the current Git branch."
-	echo "  clean           Removes all built and copied files. Use the 'all' and"
-	echo "                  'dependencies' commands to restore removed files."
-	echo "  test            Tests the default mods for errors."
+	echo "  clean           Removes all built and copied files. Use the 'all'"
+	echo "                  command to restore removed files."
+	echo "  test            Tests the mods for errors."
```

Comment:
Likewise here - this only tests the default mods, so the previous description was correct.